### DISCUSSION TITLE
完结

### DISCUSSION THREAD

**Discussion by dokieyang** [Tencent/libpag#1876] [General] [answered]

**Comment by domchen** [comment, marked as answer]
3个都不行，vap和svga的格式我们目前没有写工具支持转换，第三方可以自己实现一下，PAG的编解码都是开源的。官方提供的转换工具只支持视频，gif，apng，png序列以及lottie格式到PAG的转换。

**Reply by FoneG** [reply]
你好 可以提供下视频转Pag工具的链接吗

**Reply by domchen** [reply]
https://pag.art/docs/pag-convert-download.html

**Reply by FoneG** [reply]
链接打不开了

**Reply by domchen** [reply]
我可以打开，自己也可以从官网文档侧边栏的目录可以点进去，「转换工具下载」
